alpha/lib/model/UserLoginDataPeer.php

```diff
@@ -557,6 +557,12 @@ private static function userLogin(UserLoginData $loginData = null, $password, $p
 			}
 		}
 
+		self::setLastLoginFields($loginData, $kuser);
+		return $kuser;
```

Comment:
In case of user that was just created, its id will be generated only after calling save.
Therefor ** setLastLoginFields** must return the kuser after save.